lib/index.ts

```diff
@@ -8,7 +8,8 @@ import { IProcessInfo, IProcessTreeNode, IProcessCpuInfo } from 'windows-process
 
 export enum ProcessDataFlag {
   None = 0,
-  Memory = 1
+  Memory = 1,
+  Arguments = 2
```

Comment:
Thinking about this again we should probably name this "CommandLine" instead of Arguments as that's the Windows terminology for the exe+args.